plugins/source/aws/resources/plugin/tables.go

```diff
@@ -195,6 +195,7 @@ func tables() []*schema.Table {
 		glue.Workflows(),
 		guardduty.Detectors(),
 		iam.Accounts(),
+		iam.CredentialReportUsers(),
```

Comment:
I would call this just `iam.CredentialReport`

Reply:
Hmm, I'm just worried that the table will then be called `credential_reports`, which doesn't sound right considering that it contains user entries, not report entries. WDYT?

Reply:
Users are the only resource that have credentials though